index.html

```diff
@@ -150,6 +165,69 @@ <h3>Project name goes here</h3>
           </div>
         </div>
       </section>
+
+      <section  id='project' class="desktop-version">
```

Comment:
- [ ] As you may notice in Figma, the content of a specific project is only visible when the project image is hovered over. Kindly consider implementing that.
`Hint: You can do something like this:`
`HTML`
`<a>Hover over me!</a>`
`<div>Stuff shown on hover</div>`
`CSS`
`div {`
    `display: none;`
`}`
    
`a:hover + div {`
    `display: block;`
`}`
`NB: ALL THE IMAGES ARE PROJECTS`

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/206682850-bfd89b56-66fa-4a04-ba8b-f2130bc89cdf.png)
Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/206682941-c2db3d5a-c57a-4a11-8272-5de7e3627949.png)
![image](https://user-images.githubusercontent.com/97100091/206683036-48caac82-8602-48e2-8b77-39ae2251034e.png)